_posts/en/newsletters/2023-11-22-newsletter.md

```diff
@@ -0,0 +1,176 @@
+      use DNS and HTTPS.
+
+  Teinturier offers three proposed designs based on offers:
+
+    - _Linking domains to nodes:_ a DNS record maps a domain (e.g.
+      example.com) to an LN node identifier.  The spender sends an
+      [onion message][topic onion messages] to that node requesting an
+      offer for the ultimate receiver (e.g. alice@example.com).  The
+      domain node replies with an offer signed by its node key, allowing
+      the spender to later prove fraud if it provided an offer that
+      wasn't from Alice.  The spender can now use the offer protocol to
+      request an invoice from Alice.  The spender can also associate
+      alice@example.com with the offer, so it doesn't need to contact
+      the domain node for future payments to Alice.  Teinturier notes
+      that this design is extremely simple.
+
+    - _Certificates in node announcements:_ the existing mechanism an LN
+      node uses to advertise itself to the network is modified to allow
+      an advertisement to contain an SSL certificate chain proving that
+      (according to a certificate authority) the owner of example.com
+      claimed this particular node is controlled by alice@example.com.
+      Teinturier notes that this would require LN implementations to
+      implement SSL-compatible cryptography.
+
+    - _Store offers directly in DNS:_ a domain may have multiple DNS
+      records that directly store offers for particular addresses.  For
+      example, a DNS `TXT` record, `alice._lnaddress.domain.com`,
+      includes an offer for Alice.  Another record,
+      `bob._lnaddress.domain.com`includes an offer for Bob.  Teinturier
+      notes that this requires the domain owner create one DNS record
+      per user (and update that record if the user needs to change their
+      default offer).
+
+  As of this writing, there had not been any public discussion of the
+  ideas.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **BitMask Wallet 0.6.3 released:**
+  [BitMask][bitmask website] is a web and browser extension-based wallet for Bitcoin,
+  Lightning, RGB, and [payjoin][topic payjoin].
+
+- **Opcode documentation website announced:**
+  The [https://opcodeexplained.com/] website was recently [announced][OE tweet]
+  and provides explanations of many of Bitcoin's opcodes. The effort is ongoing
+  and [contributions are welcome][OE github].
+
+- **Athena Bitcoin adds Lightning support:**
+  The Bitcoin ATM [operator][athena website] recently [announced][athena tweet]
+  support for receiving Lightning payments for cash withdrawals.
+
+- **Blixt v0.6.9 released:**
+  The [v0.6.9][blixt v0.6.9] release includes support for simple taproot
+  channels, defaults to [bech32m][topic bech32] receive addresses, and adds
+  additional [zero conf channel][topic zero-conf channels] support.
+
+- **Durabit whitepaper announced:**
```

Comment:
Regarding the two papers in this section, I think we should keep them for this newsletter, but I would prefer in the future that we encourage the authors of ideas to post them to one of our [sources](https://bitcoinops.org/internal/sources/).  We want ideas like these to have a chance to be peer reviewed before we write about them.

Reply:
I think this is fair